db/migrate/20210308092402_add_os_minor_version_to_profiles.rb

```diff
@@ -0,0 +1,17 @@
+class AddOsMinorVersionToProfiles < ActiveRecord::Migration[5.2]
+  def up
+    remove_index :profiles, name: 'uniqueness'
+    add_column :profiles, :os_minor_version, :string, null: false, default: ''
```

Comment:
We might also want an index for searching:

```rb
add_index :profiles, :os_minor_version
```

Reply:
I guess it wouldn't matter if you filter by a lot of values (ref_id, benchmark, etc.), but if you run something like `Profile.where(os_minor_version: '7')` on all our profiles, I'd want it to be quick. We're going to expose this sort of search in our APIs, so I think an index makes sense.